.env

```diff
@@ -0,0 +1 @@
+VITE_MOVIE_API_KEY=e85d401facf2409590d6110e059c9c79
```

Comment:
This token should not be made public as anyone on github will be able to use it.  Next time, add .env to .gitignore before pushing to github.